package.json

```diff
@@ -67,6 +72,10 @@
     "mongoose-patch-history": "^2.0.0",
     "nconf": "0.10.0",
     "nodemailer": "^6.6.3",
+    "nodemon": "^2.0.20",
```

Comment:
This should be a dev dependency + would be convinient to add a command in the scripts to watch and hot-reload changes using nodemon.

Reply:
Thanks yes it should be dev dependency 👍🏼, and I already added a command to hot-reload